src/test/scala/com/danielasfregola/twitter4s/http/clients/rest/v2/tweets/TwitterV2TweetLookupClientSpec.scala

```diff
@@ -0,0 +1,99 @@
+package com.danielasfregola.twitter4s.http.clients.rest.v2.tweets
+
+import akka.http.scaladsl.model.HttpMethods
+import com.danielasfregola.twitter4s.entities.RatedData
+import com.danielasfregola.twitter4s.entities.v2.enums.expansions.TweetExpansions
+import com.danielasfregola.twitter4s.entities.v2.enums.fields.{MediaFields, PlaceFields, PollFields, TweetFields, UserFields}
+import com.danielasfregola.twitter4s.entities.v2.responses.TweetsResponse
+import com.danielasfregola.twitter4s.helpers.ClientSpec
+
+class TwitterV2TweetLookupClientSpec extends ClientSpec {
+
+  class TwitterV2TweetLookupClientSpecContext extends RestClientSpecContext with TwitterV2TweetLookupClient
+
+  "Twitter Tweet Lookup Client" should {
+
+    "lookup tweets" in new TwitterV2TweetLookupClientSpecContext {
+      val result: RatedData[TweetsResponse] = when(lookupTweets(Seq("123", "456")))
+        .expectRequest { request =>
+          request.method === HttpMethods.GET
+          request.uri.endpoint === "https://api.twitter.com/2/tweets"
+          request.uri.rawQueryString === Some("ids=123%2C456")
+        }
+        .respondWithRated("/twitter/rest/v2/tweets/tweetlookup/tweets.json")
+        .await
+      result.rate_limit === rateLimit
+      result.data === loadJsonAs[TweetsResponse]("/twitter/rest/v2/tweets/tweetlookup/tweets.json") // TODO setup fixtures
+    }
+
+    "lookup tweets with expansions" in new TwitterV2TweetLookupClientSpecContext {
+      val tweetIds = Seq("123","456")
+      val expansions = Seq(
+        TweetExpansions.`Entities.Mentions.Username`,
+        TweetExpansions.InReplyToUser,
+        TweetExpansions.`ReferencedTweets.Id`,
+        TweetExpansions.`ReferencedTweets.Id.AuthorId`
+      )
+
+      val result: RatedData[TweetsResponse] = when(lookupTweets(
+        ids = tweetIds,
+        expansions = expansions
+      ))
+        .expectRequest { request =>
+          request.method === HttpMethods.GET
+          request.uri.endpoint === "https://api.twitter.com/2/tweets"
+          request.uri.rawQueryString === Some(
+            "expansions=attachments.poll_ids%2Cattachments.media_keys%2Cauthor_id%2Centities.mentions.username%2Cgeo.place_id%2Cin_reply_to_user_id%2Creferenced_tweets.id%2Creferenced_tweets.id.author_id&ids=123%2C456"
+          )
+        }
+        .respondWithRated("/twitter/rest/v2/tweets/tweetlookup/tweets.json")
+        .await
+      result.rate_limit === rateLimit
+      result.data === loadJsonAs[TweetsResponse]("/twitter/rest/v2/tweets/tweetlookup/tweets.json") // TODO setup fixtures
```

Comment:
I believe this TODO has been done :)

Reply:
Hey @DanielaSfregola! Thanks for popping in! Actually, would you mind explaining the intention behind the `/twitter/rest/**/*.json` and `/fixutres/rest/**/*.json` files? E.g. [twitter/user.json](https://github.com/DanielaSfregola/twitter4s/blob/24c8efae5ec2fc7ea6186575b386cef7b4b03352/src/test/resources/twitter/rest/users/user.json) and [fixtures/user.json](https://github.com/DanielaSfregola/twitter4s/blob/24c8efae5ec2fc7ea6186575b386cef7b4b03352/src/test/resources/fixtures/rest/users/user.json) appaer nearly identical. Would these files only differ if the structure of our deserailized model was different from what Twitter is serving?

Reply:
Correct: one is used to simulate the response from Twitter, the other asserts the information we parse (which may not necessarily be the same)